docs/changes.xml

```diff
@@ -166,6 +174,13 @@ NGINX Unit updated to 1.31.0.
          date="2023-08-31" time="18:00:00 +0300"
          packager="Nginx Packaging &lt;nginx-packaging@f5.com&gt;">
 
+<change type="feature">
+<para>
+added unit section to /status endpoint.
+unit section is about web-server version, control_socket and etc.
+</para>
+</change>
+
 <change type="change">
 <para>
 if building with njs, version 0.8.0 or later is now required.
```

Comment:
And here's the other one.

I would also prefer the changelog entry to be in its own commit. This can give things like git-revert(1) a better chance of working if we ever need to for some reason...